assets/css/admin.scss

```diff
@@ -194,17 +193,21 @@
 	}
 
 	.addons-column {
-		flex: 1 1 200px;
-		margin: 0 1%;
-		width: 49.5%;
+		flex: 1;
+		margin-right: 1em;
```

Comment:
<img width="336" alt="screen shot 2017-12-08 at 8 59 07 am" src="https://user-images.githubusercontent.com/6817400/33768941-196caa82-dbf6-11e7-9038-7ccebe1b1983.png">

This breaks mobile styling once everything goes vertical.